examples/logto/next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
```

Comment:
This file is in `.gitignore`, so should not be in version control. :thinking: Could you make sure it's removed?

Reply:
updated